core/javatests/androidx/test/core/app/ActivityScenarioTest.java

```diff
@@ -417,6 +418,8 @@ public void perform(RecreationRecordingActivity activity) {
   public void launch_callbackSequence() {
     ActivityScenario<RecordingActivity> activityScenario =
         ActivityScenario.launch(RecordingActivity.class);
+    Espresso.onIdle();
+    Espresso.onIdle();
```

Comment:
@brettchabot Does this mean that every user of the scenario needs to `onIdle` before to make sure it's resumed and focused in a non-flaky way?